zipline/data/data_portal.py

```diff
@@ -157,6 +158,13 @@ def __init__(self,
         # days data starts.
         self._day_offsets = None
 
+    @property
+    def minute_bar_reader(self):
```

Comment:
should we keep the reader internal?  there are no consumers of self.`_minute_bar_reader` outside of the dataportal.